calculator.js

```diff
@@ -58,6 +61,16 @@ const operatorPush = (buttonPushed) => {
         displayElem.innerText = 'ERROR';
     return operator;
 }
+// --FUNCTION TO CHECK OPERATOR AND RETURN ERROR IF OPERATOR NOT IN OBJECT KEY
+const operatorCheck = (operator) => {
+    //adding multiple comparisons breaks this
+    if (operator !== "subtract"){
+        console.log (operator)
+        console.log (typeof operator);
+        displayElem.innerText = 'ERROR';
+    }else
```

Comment:
You have good instincts here! You made a function called `operatorCheck` to check if the operator is in the `MATH_FUNCS`. Smart. 

In this case, you don't need to go that far, I don't think. I'll put some recs up above.